src/Core/L10n.php

```diff
@@ -297,16 +299,20 @@ public function tt(string $singular, string $plural, int $count)
 					$i = $this->stringPluralSelectDefault($count);
 				}
 
-				// for some languages there is only a single array item
-				if (!isset($t[$i])) {
+				if (isset($t[$i])) {
+					$s = $t[$i];
+				} elseif (count($t) > 0) {
+					// for some languages there is only a single array item
 					$s = $t[0];
 				} else {
-					$s = $t[$i];
+					$this->logger->warning('Found empty strings array.', ['singular' => $singular, 'plural' => $plural, 'array' => $t]);
```

Comment:
Empty strings array are intended to make strings files smaller when there's no translation.